astroquery/jplhorizons/tests/test_jplhorizons.py

```diff
@@ -198,7 +198,7 @@ def test_elements_query_payload():
             ('REF_SYSTEM', 'J2000'),
             ('REF_PLANE', 'ECLIPTIC'),
             ('TP_TYPE', 'ABSOLUTE'),
-            ('EXTRA_PREC', 'NO')
+            ('EXTRA_PREC', 'NO'),
```

Comment:
Shouldn't this have been a syntax error before?  How were we getting away with it?